src/overlays/actors/ovl_Obj_Bean/z_obj_bean.h

```diff
@@ -16,13 +21,41 @@ enum {
 };
 
 typedef struct ObjBean {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x18];
-    /* 0x015C */ ObjBeanActionFunc actionFunc;
-    /* 0x0160 */ char unk_160[0x80];
-    /* 0x01E0 */ s8 unk_1E0;
-    /* 0x01E4 */ s32 unk_1E4;
-    /* 0x01E8 */ char unk_1E8[0x1C];
+    /* 0x000 */ DynaPolyActor dyna;
+    /* 0x15C */ ObjBeanActionFunc actionFunc;
+    /* 0x160 */ ColliderCylinder collider;
+    /* 0x1AC */ s16 unk_1AC;
+    /* 0x1AE */ s16 unk_1AE;
+    /* 0x1B0 */ s16 unk_1B0;
+    /* 0x1B2 */ s16 unk_1B2;
+    /* 0x1B4 */ s8 unk_1B4;
+    /* 0x1B8 */ f32 unk_1B8;
+    /* 0x1BC */ Vec3f unk_1BC;
+    /* 0x1C8 */ f32 unk_1C8;
+    /* 0x1CC */ f32 unk_1CC;
+    /* 0x1D0 */ f32 unk_1D0;
+    /* 0x1D4 */ Vec3s* unk_1D4;
+    /* 0x1D8 */ s16 unk_1D8;
+    /* 0x1DA */ s16 unk_1DA;
+    /* 0x1DC */ s16 unk_1DC;
+    /* 0x1DE */ s8 unk_1DE;
+    /* 0x1DF */ s8 unk_1DF;
+    /* 0x1E0 */ s8 unk_1E0;
+    /* 0x1E1 */ char unk1E1[3];
+    /* 0x1E4 */ s32 unk_1E4;
```

Comment:
This looks like padding. I think you should be able to remove it
```suggestion
    /* 0x1E0 */ s8 unk_1E0;
    /* 0x1E4 */ s32 unk_1E4;
```